tsconfig.json

```diff
@@ -18,6 +18,7 @@
     "strictNullChecks": true,
     "noUnusedLocals": true,
     "allowSyntheticDefaultImports": true,
+    "types": ["jest"],
```

Comment:
This prevents all namespaces (globals) from being automatically imported. Cypress includes mocha and chai which are also namespaces and conflict with each other.